ui/app/routes/vault/cluster/clients.ts

```diff
@@ -30,14 +37,8 @@ export default class ClientsRoute extends Route {
   model() {
     // swallow config error so activity can show if no config permissions
     return hash({
-      config: this.store.queryRecord('clients/config', {}).catch(() => {}),
+      config: this.store.queryRecord('clients/config', {}).catch(() => null),
```

Comment:
Out of curiosity - why update this from `{}` to `null`?

Reply:
When I converted to ts it was flagging this because it's not actually returning an object but rather it's an empty arrow function which is not allowed so that's why I changed it to return null. I'll add a test for the page component to make sure nothing errors out if their is no config passed in.

Reply:
Cool - thanks for explaining!